lib/include/cne_strings.h

```diff
@@ -260,6 +260,20 @@ cne_strqtok(char *str, const char *delim, char *argv[], int maxtokens)
     return argc;
 }
 
+/**
+ * Convert characters in \p str to lowercase.
+ *
+ * @param str
+ *   String to convert to lowercase
+ */
+static __inline__ void
+cne_strtolower(char *str)
+{
+    for (int i = 0; i <= (int)(strlen(str)); i++) {
```

Comment:
Single line for or if statements do not need {}

Reply:
oops - I missed this I will fix that